ports/asylum/Asylum.sh

```diff
@@ -35,8 +33,10 @@ export XDG_DATA_HOME="$CONFDIR"
 
 $ESUDO chmod 666 /dev/uinput
 
+# We need to wait until bind_files comes out of beta, for now it will remain the same.
 $ESUDO rm -rf ~/.asylum
 ln -sfv /$directory/ports/asylum/conf/.asylum ~/
+# bind_files ~/.asylum /$directory/ports/asylum/conf/.asylum
```

Comment:
Requires stable release before merge.

Reply:
Since `bind_files` is now in the stable release, these changes are ready for release.